index.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+require('vendor/autoload.php');
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+use Nexmo\Client;
+use Nexmo\Client\Credentials\Keypair;
+use Aws\S3\S3Client;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+use Aws\TranscribeService\TranscribeServiceClient;
+
+Dotenv\Dotenv::create(__DIR__)->load();
+
+$app = AppFactory::create();
+
+/**
+ * Answer the call and prompt using NCCO flow
+ */
+$app->any('/webhooks/answer', function (Request $request, Response $response) {
+    $uri = $request->getUri();
+
+    if ($request->getMethod() != 'GET') {
```

Comment:
Couldn't we use `$app->get(` on line 22 instead?

Reply:
The reasoning for doing it this way is to provide back a meaningful HTTP return type versus a standard PHP error.